packages/backend/src/services/fanart.ts

```diff
@@ -0,0 +1,76 @@
+import { FanartDoc, FanartInterface, FanartModel } from '@model/fanart'
+
+/**
+ * Get all fanart
+ * @returns array of fanart document
+ */
+export const getAllFanarts = async (
+  lastId: string,
+  limit: number
+): Promise<FanartDoc[]> => {
+  // if lastId is not given or given an empty string
+  if (lastId === 'NULL' || lastId === '') {
+    return await FanartModel.find().limit(limit)
+  }
+  return await FanartModel.find({ _id: { $gt: lastId } }).limit(limit)
+}
+
+/**
+ * Create new fanart
+ * @param creator fanart's creator
+ * @param imageUrl fanart's content
+ * @returns new fanart document
+ */
+export const storeFanart = async (
+  creator: string,
+  imageUrl: string
+): Promise<FanartDoc> => {
+  const data: FanartInterface = {
+    creator,
+    imageUrl,
+    isVerified: false,
+  }
+
+  // Create new fanart
+  const fanart = new FanartModel(data)
+  await fanart.save()
+
+  return fanart
+}
+
+/**
+ * Update existing fanart
+ * @param _id fanart's id
+ * @param creator fanart's creator
+ * @param imageUrl fanart's imageUrl
+ * @param isVerified fanart's isVerified
+ * @returns new fanart document
+ */
+export const updateFanart = async (
+  _id: string,
+  creator: string,
+  imageUrl: string,
+  isVerified: boolean
+): Promise<FanartDoc> => {
+  const fanart: FanartDoc | null = await FanartModel.findById(_id).exec()
+  // if not found
+  if (!fanart) {
+    if (!fanart) throw new TypeError('fanart not found')
+  }
```

Comment:
Correct me if I'm wrong but is this a typo?

Reply:
yeah, this is a typo lol